.size-limit

```diff
@@ -1,8 +1,7 @@
 [
   {
     name: "Fundamental-React Size",
-    webpack: true,
-    path: "lib/index.js",
-    limit: "50 KB"
+    config: "config/webpack.config.sizeLimit.js",
+    limit: "550 KB"
```

Comment:
whoa. sad.

Reply:
did you read the description? 😈 

50kb was gzipped, using the new custom webpack config I haven't been able to get it to return gzipped size to size-limit, so it is reporting the parsed size. While we're now at double the gzipped size (including the peer dependency weirdness).

We're really only 12kb larger than we were previously.